packages/astro/package.json

```diff
@@ -30,20 +30,20 @@
     "dev": "tsup --watch --onSuccess \"npm run build:dts\"",
     "build": "tsup --onSuccess \"npm run build:dts\" && npm run copy:components",
     "build:dts": "tsc --emitDeclarationOnly --declaration",
-    "copy:components": "rm -rf ./components && mkdir -p ./components/ && cp  -r ./src/astro-components/* ./components/",
+    "copy:components": "rm -rf ./components && mkdir -p ./components/ && cp  -r ./src/astro-components/* ./components/ && cp ./src/types.ts ./",
```

Comment:
There's no current way to import the `Protect` type prop to our Protect components since we just copy them and it'll have the incorrect path when published to NPM. We're including the types file so that we can reuse the `Protect` type prop across our astro and react Protect components

Reply:
Shall we add `/src/types.ts` to `.gititgnore` ?